src/main/python/treccar/trec_car_classes.py

```diff
@@ -0,0 +1,643 @@
+# Use python 3.5 or higher
```

Comment:
Should we be consistent with either `treccar` or `trec_car`? For example, if the directory name is `treccar` then should this class simply be called `treccar`?

Reply:
Good point. I will change the name of the directory because it is easy to read trec_car than treccar.